SCons/Builder.py

```diff
@@ -613,6 +613,10 @@ def _execute(self, env, target, source, overwarn={}, executor_kw={}):
             t.set_executor(executor)
             t.set_explicit(self.is_explicit)
 
+        if env.get("SCONF_NODE"):
+            for node in tlist + slist:
+                node.attributes.conftest_node = 1
```

Comment:
Thinking about this more, is there a case where a user would have non conftest file as an input to a conftest? maybe the slist should not be included in this loop?

Reply:
I added slist to make sure intermediates were marked but, the execute will be called recursively to build intermediates so those intermediates should be targets in the tlist at some point.

Reply:
Yes. I think tlist should be sufficient.  slist could include header files external to the build which could change and should cause a rebuild?

Reply:
slist removed